src/scenarios/blazorlocalized/src/BlazorLocalized/wwwroot/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
nit: indents on this and `<body>`

Reply:
This is automatically generated, all templates have it without indents. See https://github.com/dotnet/performance/blob/73a1e1dfea961f2bac4e1f13c91e1a92b8455f7d/src/scenarios/blazorpizza/src/BlazingPizza.Client/wwwroot/index.html#L3